hs/Css/Timetable.hs

```diff
@@ -178,13 +178,11 @@ courseSelectCSS = do
         margin0
         padding0
         height100
-        overflow hidden
         backgroundColor purple8
         color white
     "#course-select" ? do
-        padding0
-        margin (px 18) 0 0 (px 17)
-        width100
+        padding (px 18) 0 0 (px 17)
+        margin0
         height100
         alignCenter
         overflowY scroll
```

Comment:
@david-yz-liu the scrollbar adds to the width of the div (not sure if there's a way to fix it without Javascript), so the sidebar would constantly be resizing if someone is expanding/hiding a course repeatedly. If it's not an issue then I can change it to be `auto` instead